resolver/src/hosts.rs

```diff
@@ -0,0 +1,121 @@
+use std::collections::HashMap;
+use std::fs;
+use std::time::{Duration, SystemTime};
+use std::io::{BufRead, BufReader};
+use std::fs::File;
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+use std::sync::Mutex;
+
+lazy_static! {
```

Comment:
By putting these things in as statics it will make for a challenge in how we test the software. I'm trying to decide if there's a better location where this could be placed for making testing the mappings simpler.